src/lib/components/composites/settings/project-settings/general/ProjectNameSettings.svelte

```diff
@@ -0,0 +1,121 @@
+<script lang="ts">
+    import SettingsSection from "$lib/components/composites/settings/SettingsSection.svelte";
+    import Input from "$lib/components/composites/input/Input.svelte";
+    import { Button } from "$lib/components/primitives/button";
+    import { backendService } from "$lib/grpc-api";
+    import { onMount } from "svelte";
+    import { Project } from "$lib/model/api/project";
+    import { Schema } from "$lib/schemas";
+    import { generateFieldMask } from "protobuf-fieldmask";
+    import type { ApiError } from "$lib/model/general";
+    import ErrorAlert from "$lib/components/composites/utils/ErrorAlert.svelte";
+    import { invalidate } from "$app/navigation";
+
+    interface Props {
+        projectId: string;
+        loadingProject: Promise<Project>;
+    }
+
+    const { projectId, loadingProject }: Props = $props();
+
+    let projectName: string = $state("");
+
+    let projectNameInput: Input;
+
+    let updateProjectError: ApiError | undefined = $state(undefined);
+
+    /**
+     * Loading the project name on mount.
+     */
+    onMount(async () => {
+        loadingProject
+            .then((project) => {
+                projectName = project.name;
+            })
+            .catch((error) => {
+                updateProjectError = {
+                    errorTitle: "Something went wrong while loading the project name on mount.",
```

Comment:
The user doesn't know what on mount is and they don't need to.
```suggestion
                    errorTitle: "Something went wrong while loading the project name.",
```